privacy_evaluator/models/torch/cnn.py

```diff
@@ -23,56 +29,67 @@ def ConvNet(
         )
 
 
+class ConvBlock(nn.Module):
+    def __init__(self, in_channels, out_channels):
```

Comment:
maybe we can specify the type for `in_channels` and `out_channels`, from the conv-layer we could say the two parameters are two `int`s instead of two lists of int, right?

Reply:
Good point, also more documentation is necessary in cnn.py